# Fix building with use_custom_libcxx=true
# Currently, it removes the use of -nostdlib++, which isn't available in
# clang versions < 6

--- a/build/config/posix/BUILD.gn
+++ b/build/config/posix/BUILD.gn
@@ -58,22 +58,18 @@ config("runtime_library") {
     }
 
     # Make sure we don't link against libc++ or libstdc++.
-    if (is_clang) {
-      ldflags += [ "-nostdlib++" ]
-    } else {
-      ldflags += [ "-nodefaultlibs" ]
+    ldflags += [ "-nodefaultlibs" ]
 
-      # Unfortunately, there's no way to disable linking against just libc++
-      # (gcc doesn't have -notstdlib++:
-      # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=83931); -nodefaultlibs
-      # removes all of the default libraries, so add back the ones that we need.
-      libs += [
-        "c",
-        "gcc_s",
-        "m",
-        "rt",
-      ]
-    }
+    # Unfortunately, there's no way to disable linking against just libc++
+    # (gcc doesn't have -notstdlib++:
+    # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=83931); -nodefaultlibs
+    # removes all of the default libraries, so add back the ones that we need.
+    libs += [
+      "c",
+      "gcc_s",
+      "m",
+      "rt",
+    ]
   }
 
   if (!is_mac && !is_ios && sysroot != "") {
